assets/app/scripts/directives/nav.js

```diff
@@ -27,7 +27,27 @@ angular.module('openshiftConsole')
       link: function ($scope, element, attrs) {
         var select = $('.selectpicker', element);
 
-        var updateOptions = function(projects) {
+        var updateOptions = function() {
+          // Wait until everything we need is set.
+          if (!$scope.projects ||
```

Comment:
actually, if we have `$scope.project` (which we will), we want to show that in the dropdown while we're waiting for the projects list to come back, instead of "No value"